server/http.go

```diff
@@ -915,3 +984,45 @@ func (p *Plugin) slackAttachmentToUpdatePMI(currentValue, channelID string) *mod
 
 	return &slackAttachment
 }
+
+func (p *Plugin) checkChannelPreference(channelID string) (bool, int, error) {
+	channel, appErr := p.API.GetChannel(channelID)
+	if appErr != nil {
+		return false, http.StatusInternalServerError, errors.New(appErr.Message)
+	}
+
+	zoomChannelSettingsMap, err := p.listZoomChannelSettings()
+	if err != nil {
+		return false, http.StatusInternalServerError, err
+	}
+
+	val, exist := zoomChannelSettingsMap[channelID]
+	preference := false
+	/*
+		Check if zoom settings for current channel exist.
+		Check if creating meeting is disabled in the plugin configuration.
+	*/
+	if exist {
+		if val.Preference == ZoomChannelPreferences[DefaultPreference] {
```

Comment:
`DefaultPreference` seems a bit vague for a name for this value. Maybe `DefaultChannelRestrictionPreference`?